guide/glossary/index.md

```diff
@@ -375,6 +375,12 @@ Since a [private key](#private-key) can be used to prove that the holder control
 One of the most important activities of the bitcoin network is to verify that signatures are valid.
 </div>
 
+### Signer
```

Comment:
```suggestion
### Signing device
```

Signer is too generic, as discussed in #792. We have to use either signing device or hardware wallet.